tests/media_utils/test_videos.py

```diff
@@ -251,6 +251,44 @@ def test_raises_for_bad_file(self, bad_video):
             with pytest.raises(videos.VideoCompressionError):
                 videos.compress_video(bad_video.name, vout.name, overwrite=True)
 
+    def test_default_compression_works_webm(self, high_res_video_webm):
+        with TempFile(suffix=".webm") as vout:
+            videos.compress_video(high_res_video_webm.name, vout.name, overwrite=True)
```

Comment:
are we always overwriting when we do this? i'm asking because for all of the tests overwrite is set to true. and I don't know if that's relevant but it just was notable to me

Reply:
Not always - we only do this for convenience of using a tempfile for the output (and because the TempFile context manager creates the file, we have to allow overwrite).